functions-methods.js

```diff
@@ -33,4 +55,19 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email){
+    if (email.includes(',') === false && email.includes('@') && email.charAt(email.length - 1) !== '.'){
+        console.log("true");
+    }
+    else{
+        console.log("false");
+    }
```

Comment:
Hier kun je het resultaat meteen teruggeven óf loggen:
```suggestion
    console.log(!email.includes(',') && email.includes('@') && email.charAt(email.length - 1) !== '.');
```